package.json

```diff
@@ -2,17 +2,21 @@
   "name": "remark-mermaidjs",
   "version": "3.0.0",
```

Comment:
maybe some sort of version forwarding is appropriate?

Reply:
Versioning is done in a separate commit using `npm version`.